src/cpp/session/modules/SessionFonts.hpp

```diff
@@ -0,0 +1,38 @@
+/*
+ * SessionFonts.hpp
+ *
+ * Copyright (C) 2009-20 by RStudio, PBC
```

Comment:
ultrai-nit: Most new files use `Copyright (c) 2020 by RStudio, PBC.` and leave out the `2009-`